### PR TITLE
Add deploying logic to pom.xml

### PR DESCRIPTION
**Pull request by devmehtasynopsys**
This is the deployment logic which needs to be added to pom.xml in order to get the snapshots and for release activities.

### CODE REVIEW

**Review by zaman-akib — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -109,11 +109,58 @@
...
+  <profiles>
+    <profile>
+      <id>snapshot-deployment</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <distributionManagement>
+        <snapshotRepository>
+          <id>bds-artifactory</id>
+          <url>https://artifactory.internal.synopsys.com/artifactory/bds-integrations-snapshot</url>
```
Comment (zaman-akib) [inline pom.xml:121]: Should we expose synopsys internal artifactory URLs?

@dmamidisynopsys

Comment (dmamidisynopsys) [reply pom.xml:121]: @devmehtasynopsys We never should. Use pipeline environment variables.

Comment (dmamidisynopsys) [reply pom.xml:121]: Also, remove other gradle references. See details below under Jenkins Security Scan / security-scan / scan (pull_request)

Comment (devmehtasynopsys) [reply pom.xml:121]: will do thanks

**Review by zaman-akib — APPROVED** [review]

**Pull request merged**